.github/workflows/autoupdate.yml

```diff
@@ -179,11 +179,15 @@ jobs:
                 else # newer PRs
                   NEW_TITLE="$(echo "$OLD_TITLE" | cut --complement -f 7 -d ' ') $(echo "$TITLE" | cut -f 7 -d ' ')"
                 fi
-                gh pr edit "$PR_NUMBER" -t "$NEW_TITLE"
+                if [ "$NEW_TITLE" != "$OLD_TITLE" ]; then
+                  gh pr edit "$PR_NUMBER" -t "$NEW_TITLE"
+                fi
                 # If the PR is closed, we need to reopen it.
                 if [ "$PR_STATUS" = "CLOSED" ]; then
                   gh pr reopen "$PR_NUMBER"
```

Comment:
I don't think reopening the PR works (at least not always), see at the end of https://github.com/planemo-autoupdate/autoupdate/actions/runs/10553401308/job/29233643651

Reply:
Indeed. I tried to change the strategy when the branch has been deleted. Would you mind to review?